executor/runtime/docker/docker.go

```diff
@@ -661,14 +671,29 @@ func prepareNetworkDriver(cfg Config, c *runtimeTypes.Container) error {
 		"--batch-size", strconv.Itoa(cfg.batchSize),
 	}
 
+	// This blocks, and ignores kills.
+	if !c.TitusInfo.GetIgnoreLaunchGuard() {
```

Comment:
If IgnoreLaunchGuard is true, why would we want to skip this step?

Reply:
If IgnoreLaunchGuard is true, we should be using the kill initiated state, so we should never have two tasks using the same interface for different security groups.